data/Omdeshkar.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Om Deshkar",
+  "type": "personal",
+  "bio": " 19 |Learning front end dev | Open source newbie",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/OmDeshkar3",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Cheak out my Github",
```

Comment:
there is typo at line 12 , it should be **check** .

Reply:
```suggestion
      "name": "Check out my Github",
```